packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -1733,6 +1872,12 @@ All groups have the following properties:
 
 > **Note:** The `id`, `created`, `lastUpdated`, `lastMembershipUpdated`, `objectClass`, `type`, and `_links` properties are available only after you create a Group.
 
+In addition, groups with type `APP_GROUP` also have the following properties:
+
+| Property                              | Description                                                                                     | DataType                   | Nullable | Unique | Readonly | MinLength | MaxLength | Validation |
+| ------------------------------------- | ----------------------------------------------------------------------------------------------- | -------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
+| source <ApiLifecycle access="ea" />   | the ID of the source [application](/docs/reference/api/apps/#application-object) of the group.  | Array of String            | FALSE    | FALSE  | TRUE     |           |           |            |
```

Comment:
Nit: Capitalize `the` near `the ID of the source`

Reply:
Well, the change a few lines down _does_ capitalize the `The`.  Is that part also consistent with the rest of its form?

Reply:
@daytonwilliams-okta and @chenningzhang-okta: entries in tables are supposed to be sentence fragments, which don't require a period. I would start these fragments with a capital letter.

The only time we have periods is if there are a mix of sentence fragments and complete sentences provided in these descriptions.